.github/workflows/build_SRWA.yml

```diff
@@ -0,0 +1,89 @@
+# This workflow Builds the UFS SRWEATHER APP
+name: Build SRWA
+
+on:
+  push:
+    branches: [ feature/CI_build_test ]
+  pull_request:
+    branches: [ rrfs_ci, feature/CI_build_test ]
+
+# Use a default login shell which loads intel/oneapi/setvars.sh, and lmod path 
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+ 
+  Clone_rrfs_ci_pcluster:
+    # Auxillary repository needed to insert env files to run CI tests in the short term 
+    name: Clone RRFS CI repo
+    # This job will run on AWS Parallel Cluster head node
+    runs-on: self-hosted
```

Comment:
@robgonzalezpita I assume the failure on this task means that we need to work together to get the self-hosted runner set up on this branch.

Reply:
The failure is due to not having a self-hosted runner set up for this repository. Yes, we do need to work together on configuring a runner to pick up jobs initiated by this workflow.